### PR TITLE
Add Tensorboard logger

### PR DESCRIPTION
**Pull request by colibrisson**
Add Tensorboard support as proposed in [mittagessen#422](https://github.com/mittagessen/kraken/issues/422).

Tensorboard logging is triggered by adding `--debug` flag to the `ketos train` command. The metrics logged are training loss, validation accuracy, and learning rate.

### CODE REVIEW

**Comment by mittagessen** [comment]
Are you sure you don't want to log the validation metric as well? It is used for certain decisions like the early stopping and the learning rate reduction on plateau so it might be useful to log it in addition to the raw accuracy.

In addition I would like to add it to the segmentation and pretraining as well.
